layouts/partials/tabs.html

```diff
@@ -60,9 +60,47 @@ <h3 class="subtitle is-5 is-muted">{{ $subtitle }}</h3>
+                       <div><a href="https://github.com/yhat/ggpy"><img src="images/content_images/v_ggpy.png" alt="visualization figure" align="middle" style="border-radius: 10px"></a></div>
+                       <div><a href="https://www.journaldev.com/19692/python-plotly-tutorial"><img src="images/content_images/v_plotly.png" alt="visualization figure" align="middle" style="border-radius: 10px"></a></div>
+                       <div><a href="https://altair-viz.github.io/gallery/streamgraph.html "><img src="images/content_images/v_altair.png" alt="visualization figure" align="middle" style="border-radius: 10px"></a></div>
+                       <div><a href="https://seaborn.pydata.org"><img src="images/content_images/v_seaborn.png" alt="visualization figure" align="middle" style="border-radius: 10px"></a></div>
+                       <div><a href="https://demo.bokeh.org/surface3d"><img src="images/content_images/v_bokeh.png" alt="visualization figure" align="middle" style="border-radius: 10px"></a></div>
+                       <div><a href="https://napari.org"><img src="images/content_images/v_napari.png" alt="visualization figure" align="middle" style="border-radius: 10px"></a></div>
+                       <div><a href="http://vispy.org/gallery.html"><img src="images/content_images/v_vispy.png" alt="visualization figure" align="middle" style="border-radius: 10px"></a></div>
+
+                     </div> 
+                    </p>
+                  </div>
+                  <div>
+                     <p>
+                     </p>
+                     <p>
+                       <a href="https://www.slideshare.net/Visage/data-visualization-101-how-to-design-chartsandgraphs">Data Visualization</a> exposes patterns, trends and correlations in textual-data, making it easier for humans to analyse and interpret large volumes of data. 
+                     </p>
+                     <p>
+                       <a href="https://python-graph-gallery.com">Visualization elements</a> such as bar graphs, pie charts, line charts, maps, infographics, dashboards, geographic maps, heatmaps, and interactive images offer valuable insights for making data-driven decisions.
+                     </p>
+                  </div>
+                  <div>
+                     <p>
+                       NumPy is the key data transformation building block for the burgeoning <a href="https://pyviz.org/overviews/index.html">Python visualization landscape</a> comprising of <a href="https://matplotlib.org">Matplotlib</a>, <a href="https://seaborn.pydata.org">Seaborn</a>, <a href="https://plot.ly">Plotly</a>, <a href="https://altair-viz.github.io">Altair</a>, <a href="https://docs.bokeh.org/en/latest/">Bokeh</a>, <a href="https://github.com/yhat/ggpy">ggplot</a>, <a href="http://vispy.org">Vispy</a> and <a href="https://github.com/napari/napari">Napari</a>, to name a few. 
```

Comment:
`ggpy` is no longer developed and has few users I think. Can you replace it with [Holoviz](http://holoviz.org/)? That's very actively developed and growing in popularity (with Panel, Datashader, etc. - mentioning just Holoviz should be fine since it's a family of packages). 

After that change is made I think the selection of packages and images is good to go.

Reply:
Done - See ref: https://github.com/numpy/numpy.org/pull/64/commits/49b73489db4985d2e75d59e5d735adec5eecd2ce